source/frontend/src/features/leases/detail/LeasePages/details/LeaseTeamView.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { FaExternalLinkAlt } from 'react-icons/fa';
+
+import { Section } from '@/components/common/Section/Section';
+import { SectionField } from '@/components/common/Section/SectionField';
+import { StyledLink } from '@/components/maps/leaflet/LayerPopup/styles';
+import { ApiGen_Concepts_Lease } from '@/models/api/generated/ApiGen_Concepts_Lease';
+import { formatApiPersonNames } from '@/utils/personUtils';
+
+export interface ILeaseTeamProps {
+  lease: ApiGen_Concepts_Lease;
+}
+export const LeaseTeamView: React.FunctionComponent<React.PropsWithChildren<ILeaseTeamProps>> = ({
```

Comment:
This looks reusable given it is iterating over a team member list. Similar to Acq team members